actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/actor-util/src/main/java/org/sunbird/actorutil/location/impl/LocationClientImpl.java

```diff
@@ -152,6 +151,24 @@ public void updateLocation(
     return new ArrayList<>();
   }
 
+  private Object actorCall(ActorRef actorRef, Request request, RequestContext context){
+    Object obj = null;
+    try {
+      Timeout t = new Timeout(Duration.create(10, TimeUnit.SECONDS));
+      Future<Object> future = Patterns.ask(actorRef, request, t);
+      obj = Await.result(future, t.duration());
+      checkLocationResponseForException(obj);
```

Comment:
This check not required.

Reply:
Here we are checking the response obj type and throwing the exception, so thinking to keep it for now.